### DISCUSSION TITLE
Cellular Hotspots

### DISCUSSION THREAD

**Discussion by robroy90** [Ysurac/openmptcprouter#2360] [Q&A]
Greetings all!

I want to set up OMR with Starlink and a pair of cellular hotspot devices. The issue is that the two cellular hotspot devices do not have ethernet ports, only USB. Will I have any issues connecting the hotspots via USB? Will OMR detect them properly when connected to the USB ports or will I need to adjust settings somehow in OMR to specify the interfaces? Thanks!

**Comment by Chris5342** [comment]
Did you get it going ?
